ibm_catalog.json

```diff
@@ -0,0 +1,185 @@
+{
+  "products": [
+    {
+      "name": "deploy-arch-ibm-watsonx-assistant",
+      "label": "watsonx Assistant",
+      "product_kind": "solution",
+      "tags": [
+        "ibm_created",
+        "target_terraform",
+        "terraform",
+        "ai",
+        "chat",
+        "solution"
+      ],
+      "keywords": [
+        "watsonx assistant",
+        "conversation",
+        "assistant",
+        "IaC",
+        "infrastructure as code",
+        "terraform",
+        "solution",
+        "ai"
+      ],
+      "short_description": "Creates and configures an instance of IBM watsonx Assistant.",
+      "long_description": "This architecture supports creating and configuring an instance of watsonx Assistant.",
+      "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-assistant/blob/main/README.md",
+      "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-watsonx-assistant/main/images/watsonx-assistant.svg",
+      "provider_name": "IBM",
+      "features": [
+        {
+          "title": "Creates an instance of IBM watsonx Assistant",
+          "description": "Creates and configures an IBM watsonx Assistant instance."
+        }
+      ],
+      "flavors": [
+        {
+          "label": "Standard",
+          "name": "standard",
+          "install_type": "fullstack",
+          "working_directory": "solutions/standard",
+          "compliance": {
+            "authority": "scc-v3",
+            "profiles": [
+              {
+                "profile_name": "IBM Cloud Framework for AI Security Guardrails 2.0",
+                "profile_version": "1.6.0"
```

Comment:
I think...version should be 2.0
Please check the latest version in SCC once

Reply:
There are two supported versions for this Profile. Added the latest one.
<img width="740" alt="image" src="https://github.com/user-attachments/assets/c22ddf80-1baf-4e22-a2ab-b354718d0b89" />